static/src/javascripts/projects/commercial/modules/comscore.js

```diff
@@ -0,0 +1,47 @@
+// @flow strict
+import config from 'lib/config';
+
+type comscoreGlobals = { c1: string, c2: string, comscorekw?: string };
+
+const comscoreSrc = 'https://sb.scorecardresearch.com/beacon.js';
+const comscoreC1 = '2';
+const comscoreC2 = '6035250';
+
+const getGlobals = (keywords: string): comscoreGlobals => {
+    const globals: comscoreGlobals = { c1: comscoreC1, c2: comscoreC2 };
+
+    if (keywords !== 'Network Front') {
+        globals.comscorekw = keywords;
+    }
+
+    return globals;
+};
+
+export const init = (): Promise<void> => {
+    if (config.get('switches.comscore', false)) {
```

Comment:
If this is remaining outside of the third-party-tags, can we add the same checks so it doesn't load on identity or the secure contact pages?

```js
        const isIdentityPage =
            config.get('page.contentType') === 'Identity' ||
            config.get('page.section') === 'identity';
```

```js
        const isSecureContact = [
            'help/ng-interactive/2017/mar/17/contact-the-guardian-securely',
            'help/2016/sep/19/how-to-contact-the-guardian-securely',
        ].includes(config.get('page.pageId', ''));
```

Reply:
👍